File: csharp/standardization/Algorithm_Basic.cs
using System;
using System.Collections.Generic;
using Algo.Devtools;
using AlgorithmiaPipe;

namespace Algo
{
        public class AlgorithmBasic
        {
            public static string Foo(string input)
            {
                return $"Hello {input}";
            }

            public static void Main(string[] args)
            {
                var algo = new AlgorithmHandler<String, Dictionary<String, Object>, String>(Foo);
                algo.Run();
            }
    }
}

File: java/standardization/src/main/java/com/algorithmia/example/AlgorithmBasic.java
package com.algorithmia.example;
import com.algorithmia.algorithmHandler.*;
import com.algorithmia.*;

class AlgorithmBasic
{
    String Apply(String input) throws Exception{
        return "hello ".concat(input);
    }

    public static void main(String[] args) throws Exception{
        AlgorithmBasic defs = new AlgorithmBasic();
        AlgorithmHandler algo = new AlgorithmHandler<>(defs::Apply);
        algo.run();
    }
}
